bucket/libreoffice.json

```
@@ -56,7 +56,7 @@
         "regex": ">([\\d.]+)/<"
     },
     "autoupdate": {
-        "url": "https://downloadarchive.documentfoundation.org/libreoffice/old/$version.2/portable/LibreOfficePortable_$version_MultilingualStandard.paf.exe#/dl.7z",
+        "url": "http://download.documentfoundation.org/libreoffice/portable/$version/LibreOfficePortable_$version_MultilingualStandard.paf.exe",
```

```suggestion
        "url": "http://download.documentfoundation.org/libreoffice/portable/$version/LibreOfficePortable_$version_MultilingualStandard.paf.exe#/dl.7z",
```

`#/dl.7z` means that the file will be **renamed to "dl.7z"** after downloading. (indicates that the file should be extracted using 7-zip)